quarkus/distribution/build.gradle.kts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.tools.ant.filters.ReplaceTokens
+import publishing.GenerateDigest
+
+plugins {
+    id("distribution")
+    id("signing")
+}
+
+description = "Apache Polaris Binary Distribution"
+
+val adminProject = project(":polaris-quarkus-admin")
```

Comment:
As we do this change, I would suggest to rename `polaris-quarkus-admin` and `polaris-quarkus-server` as `polaris-admin` and `polaris-server`, and also maybe rename `quarkus` folder as `dist` folder or so.

Reply:
Sounds good. I don't think we need `quarkus` within the module name. Quarkus is the way in Polaris, not a way. Adding Quarkus into name isn't necesary.

Reply:
Yes, and I think it makes sense to do it as part of this PR (for consistency).

Reply:
+1 to `polaris-admin` and `polaris-server`